Lib/unittest/case.py

```diff
@@ -385,8 +387,8 @@ class TestCase(object):
         the instance's assertion methods fail; test methods raising this
         exception will be deemed to have 'failed' rather than 'errored'.
     * longMessage: determines whether long messages (including repr of
-        objects used in assert methods) will be printed on failure in *addition*
-        to any explicit message passed.
+        objects used in assert methods) will be printed on failure in
+        *addition* to any explicit message passed.
```

Comment:
Unrelated to the main change (and also 80 chars is fine)
```suggestion
        objects used in assert methods) will be printed on failure in *addition*
        to any explicit message passed.
```